atc/db/migration/migrations/1554304383_add_aborted_to_builds.up.sql

```diff
@@ -0,0 +1,7 @@
+BEGIN;
+
+  ALTER TABLE builds ADD COLUMN aborted boolean DEFAULT false NOT NULL;
+
+  UPDATE builds SET aborted = true, completed = true WHERE status = 'aborted';
```

Comment:
:+1: this is important so that old builds that were aborted but not completed don't gum up the API and make it think it's the 'next build'